Tests/DatadogTests/Datadog/RUM/WebView/WebRUMEventConsumerTests.swift

```diff
@@ -122,5 +142,7 @@ class WebRUMEventConsumerTests: XCTestCase {
         let writtenJSON = try XCTUnwrap(try JSONSerialization.jsonObject(with: data, options: []) as? JSON)
 
         AssertDictionariesEqual(writtenJSON, unknownWebRUMEvent)
+        let webViewCommand = try XCTUnwrap(mockCommandSubscriber.lastReceivedCommand)
+        XCTAssertEqual(webViewCommand.time, .mockDecember15th2019At10AMUTC())
     }
```

Comment:
nit: instead of updating each test case with a new side effect we could simply add one new unit test with fuzzy input:
* Given: (random RUM context `x` random Web RUM event)
* When: `try eventConsumer.consume(event: webRUMEvent)`
* Then: `mockCommandSubscriber receives` "keep alive" command

This has classic benefit of fuzzy testing: covers all possible inputs for new requirement within a single test (vs now we cover new requirement in all other tests). Also, the test will be named in a way that describes new behaviour solely (vs now we don't call new behaviour from any of existing tests - the reader needs to guess it from assertions). Last, adding next behaviours to this suite will require adding only one test (vs now the convention is to test new behaviour in all existing tests).

Reply:
if that random input fails, it's very likely that we will re-run CI given that our tests are quite flaky.
i prefer keeping it deterministic in this case.